sessions/react-effects-and-localstorage/journal-app-localstorage/src/components/Entry/index.js

```diff
@@ -0,0 +1,25 @@
+import FavoriteButton from "../FavoriteButton";
+import "./Entry.css";
+
+function Entry({ motto, notes, date, id, isFavorite, onToggleFavorite }) {
+  return (
+    <article className="entry">
+      <div className="entry__date">{date}</div>
```

Comment:
I think that the HTML `<time>` element is more semantic and we can get rid of a `div` here.

Reply:
Good Idea! I changed it to be `<time>`